docker-compose.yml

```diff
@@ -30,5 +30,37 @@ services:
       - ENV=docker
       - STATIC_ROOT=/var/www/static
 
+  keycloak_db:
```

Comment:
Why aren't you just using the existing db?

Reply:
Simply to have things separated. But since we have everything in the same docker-compose I might as well use the same db